src/elements/content-uploader/UploadsManager.scss

```diff
@@ -2,9 +2,14 @@
 
 .be.bcu-uploads-manager-container {
     position: fixed;
+    right: 56px;
```

Comment:
Are these positioning styles supposed to be here or in the parent app? Only generic styles should be here ideally.
And for the box-shadow, can you use the $black variable instead. rgba( $black, .1 )

Reply:
My bad. All changes go under a deja blue class.